pysat/tests/test_utils.py

```diff
@@ -71,35 +119,49 @@ def teardown(self):
     # test pysat data dir options
     def test_set_data_dir(self):
         """update data_dir"""
+        import pysat
```

Comment:
Is it really correct to double import?  Would reload be more appropriate?

Reply:
reload is bit weird now. reload doesn't work directly because it complains pysat isn't a module. `type(pysat)` returns different information now. I had to do a workaround to get the effect of a reload, which is the `pysat.pysat_reload()` but then you have to pull in the updated module from the global namespace, `pysat = sys.modules['pysat']`. Once there is a line `pysat = something` in the code the Python interpreter starts to complain if pysat isn't explicitly defined in more places.